src/main/groovy/com/bmuschko/gradle/docker/tasks/network/DockerRemoveNetwork.groovy

```diff
@@ -0,0 +1,10 @@
+package com.bmuschko.gradle.docker.tasks.network
+
+class DockerRemoveNetwork extends DockerExistingNetwork {
+    @Override
+    void runRemoteCommand(final dockerClient) {
+        final networkId = getNetworkId()
+        logger.quiet "Removing network '$networkId'."
+        dockerClient.removeNetworkCmd(networkId).exec()
```

Comment:
hmmm ... I wonder if we should start here with setting a standard. If the user were to define `onNext` here, but we have nothing to hand off, than it's more/less a no-op. I wonder if we should pass in say `true` just to give the user some feedback. @orzeh any thoughts?